packages/fiori/src/themes/Wizard.css

```diff
@@ -52,6 +52,10 @@
 	padding-bottom: 1rem;
 }
 
+:host([renderMode="Page"]) .ui5-wiz-content-item:not([selected]) {
```

Comment:
`renderModel` property should be `render-mode` attribute

Reply:
Done.